app/code/Magento/SalesRule/Block/Adminhtml/Promo/Quote/Edit/Tab/Coupons/Grid.php

```diff
@@ -151,4 +152,16 @@ public function getGridUrl()
     {
         return $this->getUrl('sales_rule/*/couponsGrid', ['_current' => true]);
     }
+
+    /**
+     * @inheritdoc
+     */
+    public function toHtml()
```

Comment:
@edenduong, could you please verify if it's possible to solve the mentioned issue by overriding the `_toHtml` method instead of `toHtml`?
If not, then let's leave it as it is.

Thank you!

Reply:
Hello @dmytro-ch ,

I have changed the source code to _toHtml(). Please check it again.

Thanks